recipes-runtime/dotnet-core/dotnet-core_6.0.0.inc

```diff
@@ -6,10 +6,10 @@ SUMMARY = "Contains the binaries for Microsoft's .NET 6.0.0"
 HOMEPAGE = "https://dotnet.microsoft.com/download/dotnet/6.0"
 
 DOTNET_RUNTIME_ARCH = "none"
-DOTNET_RUNTIME_ARCH_arm = "arm"
-DOTNET_RUNTIME_ARCH_x86-64 = "x64"
-DOTNET_RUNTIME_ARCH_x86_64 = "x64"
-DOTNET_RUNTIME_ARCH_aarch64 = "arm64"
+DOTNET_RUNTIME_ARCH:arm = "arm"
+DOTNET_RUNTIME_ARCH:x86-64 = "x64"
+DOTNET_RUNTIME_ARCH:x86_64 = "x64"
+DOTNET_RUNTIME_ARCH:aarch64 = "arm64"
 
 # This is here because it doesn't seem like bitbake likes ${PV} used in require statements.
-require recipes-runtime/dotnet-core/dotnet-core_6.0.0_${DOTNET_RUNTIME_ARCH}.inc
```

Comment:
This will force architecture to arm.

Reply:
yeah, sorry. Made this because I didn't understand the new requirements for overloading in honister. Can submit a PR to fix it :)